scripts/deploy.ts

```diff
@@ -24,11 +27,12 @@ function getDefaultGenesisRoot(parameters: DeploymentParameters) {
 }
 
 async function main() {
-    const provider = new ethers.providers.JsonRpcProvider(argv.url);
-    const signer = provider.getSigner();
-    // const signer = new ethers.Wallet(
-    //     "0xyourprivatekeyhere"
-    // ).connect(provider);
+    const provider = new ethers.providers.JsonRpcProvider(
+        argv.url ?? "http://localhost:8545"
+    );
+    const signer = argv.key
+        ? new ethers.Wallet(argv.key).connect(provider)
+        : provider.getSigner();
```

Comment:
Beautiful! With this change, we can gracefully pass the private key to the command argument. I like it so much.